Header header

# ускорения по соответствующим осям
float32 acc_x
float32 acc_y
float32 acc_z